h/static/scripts/directive/form-input.coffee

```diff
@@ -0,0 +1,33 @@
+module.exports = ->
+  link: (scope, elem, attr, [model, validator]) ->
+    return unless  model
```

Comment:
The more I think about this, the more I'd be much happier if you could back out these second-order changes.